File: level 77/homework/hw.js
function createObject(keys, values) {
    let obj = {};
    for (let i = 0; i < keys.length; i++) {
        obj[keys[i]] = values[i];
    }
    return obj;
}

// ტესტირება
let keys = ['name', 'age', 'city'];
let values = ['Alice', 30, 'New York'];

let result = createObject(keys, values);
console.log(result);  // { name: 'Alice', age: 30, city: 'New York' }

File: level 076/homework/hw.py
def sum_digits(n):
    result = 0
    while n > 0:
        result += n % 10
        n //= 10  # შეცვლა: /=-ზე
    return result

print(sum_digits(123))  # მოსალოდნელი შედეგი: 6 (1 + 2 + 3)
print(sum_digits(405))  # მოსალოდნელი შედეგი: 9 (4 + 0 + 5)
